var/spack/repos/builtin/packages/sherpa/package.py

```diff
@@ -0,0 +1,129 @@
+    version('1.3.0', sha256='08b13c65b66f2edde6996d2a06762a12a0682ffb64bca43654df47321e5039a0')
+    version('1.2.3', sha256='029727337a430d6675a1a12dce3ced0411041e79ddaf4ce3b9466035cf6c8804')
+    version('1.2.2', sha256='6e7b5ea80b99f1378519009e494030d6cf4c4491f91218d749eabb8ffaad9ac1')
+    version('1.2.1', sha256='838462f4a1e8768135363aa6b8532fd8f5e5789a269b858f8e3728ab37f6a1d1')
+    version('1.2.0', sha256='509508fd0ad72aaf55ab484da8b6bc0b31688c955adcda62a3e8f94689cebf99')
+    version('1.1.3', sha256='6335e5eb1fc304e9618496d3ddb198b3591e57b27db6e876af8fd649a8b98c93')
+    version('1.1.2', sha256='e1689cad6700dc013af0afb0d33729ac2b5e9841d2f325c85b10d773e7f8a80e')
+    version('1.1.1', sha256='b80e1d75934be79b73400d2c95d96e88651626ea29ddcb9d8fde9c1812039e29')
+    version('1.1.0', sha256='8052d137d668353dc710f8691b921e772820d39e20361f0d616ee2da1ac798f2')
+    version('1.0.9', sha256='fe28db91ea8264364395c7e5efeeae3e5c01ea1343e0db7fe13924c6f17fb963')
+    version('1.0.8', sha256='6e346bafd13b5b05ad566a73759da6d5e64d65c5036780cc4911d93277e891fa')
+    version('1.0.7', sha256='d1eeefd96c6822ea8eb926447ca91ec4a1c714e4746323e92b1e17764e51ff0b')
+    version('1.0.6', sha256='358d417ec3afde24618c222bc9b742bc5102d435622b3cd6f2e3f72d03656255')
+
+    _cxxstd_values = ('11', '14', '17')
+    variant('cxxstd',    default='11', values=_cxxstd_values, multi=False,
+            description='Use the specified C++ standard when building')
+
+    variant('analysis',   default=True, description='Enable analysis components')
+    variant('mpi',        default=False, description='Enable MPI')
+    variant('python',     default=False, description='Enable Python API')
+    variant('hepmc2',     default=True, description='Enable HepMC (version 2.x) support')
+    variant('hepmc3',     default=True, description='Enable HepMC (version 3.x) support')
+    variant('hepmc3root', default=False, description='Enable HepMC (version 3.1+) ROOT support')
+    variant('rivet',      default=False, description='Enable Rivet support')
+    variant('fastjet',    default=True, description='Enable FASTJET')
+    variant('openloops',  default=False, description='Enable OpenLoops')
+    variant('recola',     default=False, description='Enable Recola')
+    variant('lhole',      default=False, description='Enable Les Houches One-Loop Generator interface')
+    variant('root',       default=False, description='Enable ROOT support')
+    variant('lhapdf',     default=True, description='Enable LHAPDF support')
+    variant('gzip',       default=False, description='Enable gzip support')
+    variant('pythia',     default=True, description='Enable fragmentation/decay interface to Pythia')
+
+    # Note that the delphes integration seems utterly broken: https://sherpa.hepforge.org/trac/ticket/305
+
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool',  type='build')
+    depends_on('m4',       type='build')
+    depends_on('sqlite')
```

Comment:
```suggestion
    depends_on('sqlite')
    depends_on('texinfo')
```
Ref: makeinfo needed at https://gitlab.com/sherpa-team/sherpa/-/blob/master/Manual/Makefile.am#L43.

```
make[2]: Entering directory '/home/wdconinc/.spack/stage/spack-stage-sherpa-2.2.11-et52v63lstkwey2ibxw4s2xn4t6i7jey/spack-src/Manual'
make[2]: warning: -j1 forced in makefile: resetting jobserver mode.
Updating ./version.texi
restore=: && backupdir=".am$$" && \
am__cwd=`pwd` && CDPATH="${ZSH_VERSION+.}:" && cd . && \
rm -rf $backupdir && mkdir $backupdir && \
if (/bin/bash '/home/wdconinc/.spack/stage/spack-stage-sherpa-2.2.11-et52v63lstkwey2ibxw4s2xn4t6i7jey/spack-src/missing' makeinfo --version) >/dev/null 2>&1; then \
  for f in Sherpa.info Sherpa.info-[0-9] Sherpa.info-[0-9][0-9] Sherpa.i[0-9] Sherpa.i[0-9][0-9]; do \
    if test -f $f; then mv $f $backupdir; restore=mv; else :; fi; \
  done; \
else :; fi && \
cd "$am__cwd"; \
if /bin/bash '/home/wdconinc/.spack/stage/spack-stage-sherpa-2.2.11-et52v63lstkwey2ibxw4s2xn4t6i7jey/spack-src/missing' makeinfo -I ..  -I . \
 -o Sherpa.info Sherpa.texi; \
then \
  rc=0; \
  CDPATH="${ZSH_VERSION+.}:" && cd .; \
else \
  rc=$?; \
  CDPATH="${ZSH_VERSION+.}:" && cd . && \
  $restore $backupdir/* `echo "./Sherpa.info" | sed 's|[^/]*$||'`; \
fi; \
rm -rf $backupdir; exit $rc
/home/wdconinc/.spack/stage/spack-stage-sherpa-2.2.11-et52v63lstkwey2ibxw4s2xn4t6i7jey/spack-src/missing: line 81: makeinfo: command not found
WARNING: 'makeinfo' is missing on your system.
         You should only need it if you modified a '.texi' file, or
         any other file indirectly affecting the aspect of the manual.
         You might want to install the Texinfo package:
         <https://www.gnu.org/software/texinfo/>
         The spurious makeinfo call might also be the consequence of
         using a buggy 'make' (AIX, DU, IRIX), in which case you might
         want to install GNU make:
         <https://www.gnu.org/software/make/>
make[2]: *** [Makefile:580: Sherpa.info] Error 127
make[2]: Leaving directory '/home/wdconinc/.spack/stage/spack-stage-sherpa-2.2.11-et52v63lstkwey2ibxw4s2xn4t6i7jey/spack-src/Manual'
make[1]: *** [Makefile:571: all-recursive] Error 1
make[1]: Leaving directory '/home/wdconinc/.spack/stage/spack-stage-sherpa-2.2.11-et52v63lstkwey2ibxw4s2xn4t6i7jey/spack-src'
make: *** [Makefile:497: all] Error 2
```

Reply:
Actually, just a `type='build'` dependence.

Reply:
I was trying to get around this dependency by adding `--disable-manual` to the configure arguments, but it seems that even though it is referenced, that argument is unfortunately not working.

Reply:
Let me know when this is ready to be merged

Reply:
@alalazo I added the missing dependency and reported the manual issue upstream (https://gitlab.com/sherpa-team/sherpa/-/issues/332), unless Wouter has more comments this is good to go!

Reply:
Nothing from my side. Looks good to go.